templates/article-cover.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Facebook Instant Article Cover template, containing the following info:
+ *
+ * 1) Media type: Image, Video or Slideshow
+ * 2) Title
+ * 3) Subtitle
+ * 4) Author(s)
+ * 5) Kicker (aka "A tertiary blurb in the headline of the article.")
+ * 6) Published Time
+ * 7) Modified Time
+ *
+ * @see https://developers.facebook.com/docs/instant-articles/guides/articlecreate#specify-cover
+ */
+?>
+<header>
+
+	<?php
+	// Post featured image as FB IA cover image.
+	if ( $thumb_id = get_post_thumbnail_id() ) :
+	?>
+		<figure>
+			<?php echo wp_get_attachment_image( $thumb_id, array( 2048, 2048 ) ); ?>
```

Comment:
@mattheu not sure if CI gonna complain about unescaped output. I wanted to use this function to minimise WP's filters, so that image HTML as simple as possible.